### ISSUE TITLE
.realPress("ArrowRight") is losing focus for a11y testing when combine .realPress("Tab") and .realPress("ArrowRight");

### ISSUE THREAD

**Issue by 787681277** [dmtrKovalenko/cypress-real-events#355]
cy.get(tab_tooltip)
       .realPress("Tab")
      .realPress("ArrowRight");

//working
.realPress("Tab"): This operation will focus active tab and show the tooltip in same time  

//not working
.realPress("ArrowRight"):This operation will activate neighbour tab, but lose focus effect and lose tooltip at the same time. which cause accessibility testing failed

//working
<img width="639" alt="Screen Shot 2022-12-07 at 12 36 20" src="https://user-images.githubusercontent.com/26969223/206089521-4ccbb78f-b75c-49f2-b1fc-cfd5e6fc309b.png">

//not working
<img width="1229" alt="Screen Shot 2022-12-07 at 11 55 41" src="https://user-images.githubusercontent.com/26969223/206084487-0d489432-0ccc-4821-8e64-91611f332722.png">

**Comment by dmtrKovalenko** [comment]
You will likely need to add cy.window().focus() to firstly move focus inside the tab
